R/coh_data_wrangling.R

```diff
@@ -352,9 +363,14 @@ get_time_to_event <- function(data, outcome_date_col,
     colnames(data),
     must.include = outcome_date_col
   )
-  # check strings, converted to date later
-  checkmate::assert_string(start_cohort)
-  checkmate::assert_string(end_cohort)
+  # check date types
+  checkmate::assert_date(start_cohort)
+  checkmate::assert_date(end_cohort)
```

Comment:
Check for single length and no missing values here.

Reply:
Done